src/config.ts

```diff
@@ -1,15 +1,33 @@
 import dotenv from "dotenv";
 
+export enum EnvironmentMode {
+  Local = "local",
+  Staging = "staging",
+  Production = "production",
+}
+
 dotenv.config();
 
 // AWS SDK
 
 export const AWS_REGION: string = "ca-central-1";
 
+// Environment mode
+
+export const ENVIRONMENT_MODE: EnvironmentMode =
+  mapEnvironmentModeFromStringToEnum(loadRequiredEnvVar("ENVIRONMENT_MODE"));
+
 // Express
 
 export const SERVER_PORT: number = 3001;
 
+// Freshdesk
+
+export const FRESHDESK_API_URL: string = "https://cds-snc.freshdesk.com/api";
+
+export const FRESHDESK_API_KEY: string =
+  loadRequiredEnvVar("FRESHDESK_API_KEY");
```

Comment:
TODO: Will have to update the API ECS task to add Freshdesk as an environment variable

Reply:
https://github.com/cds-snc/forms-terraform/pull/795